### DISCUSSION TITLE
Windows Hosts file causes dnscache service issues

### DISCUSSION THREAD

**Discussion by ghost** [blocklistproject/Lists#521] [Q&A]
So I made a custom hosts file using the
abuse.txt crypto.txt drugs.txt facebook.txt fraud.txt gambling.txt malware.txt phishing.txt piracy.txt porn.txt ransomware.txt torrent.txt tiktok.txt whatsapp.txt

The end file was about 105MB. So I dropped it into `C:\Windows\System32\drivers\etc\hosts` then once I did the dnscache service just started freaking out and eating up a ton of memory and CPU and the rest of the computer become unusable. Has anyone else experienced this?

**Comment by gap579137** [comment]
Windows hosts file was not made to be that big. You are going to have issues. I would recommend using adguard or such and loading in our lists.
